### ISSUE TITLE
Support searchPath (or similar)

### ISSUE THREAD

**Issue by ocdtrekkie** [zenhack/docker-spk#14]
We should be able to get things like the fake proc mounts working with docker-spk, similarly to how it is done with a normal spk or vagrant-spk package: https://github.com/zenhack/dotnet-docker-spk

Perhaps since spk is not "searching for files" here, it should work more like an alwaysInclude, and ignore the standard `.` and `\` paths. We just want the stuff people are explicitly adding with a custom packagePath.

**Comment by zenhack** [comment]
Yeah, I think we could do something where we just included the full packagePath for each entry in searchPath, in addition to what gets built in to the docker container.
